cmd/flux/create_source_git_test.go

```diff
@@ -173,6 +173,21 @@ func TestCreateSourceGit(t *testing.T) {
 				apimeta.SetStatusCondition(&repo.Status.Conditions, newCondition)
 			},
 		},
+		{
+			name:   "no args",
+			args:   "create secret git",
+			assert: assertError("name is required"),
+		},
+		{
+			name:   "source with commit",
+			args:   "create source git podinfo --url=https://github.com/stefanprodan/podinfo --commit=c88a2f41 --timeout=0s --export",
+			assert: assertGoldenFile("./testdata/create_source_git/source-git-commit.yaml"),
+		},
+		{
+			name:   "source with ref name",
+			args:   "create source git podinfo --url=https://github.com/stefanprodan/podinfo --ref-name=refs/heads/main --timeout=0s --export",
+			assert: assertGoldenFile("testdata/create_source_git/source-git-refname.yaml"),
+		},
```

Comment:
please move these test cases to the `TestCreateSourceGitExport` test case above. this test cases is meant for testing the reconciliation status and the resulting message